src/lib/is_tap.js

```diff
@@ -0,0 +1,16 @@
+const euclideanDistance = require('./euclidean_distance');
+
+const TOLERANCE = 25;
+const INTERVAL = 250;
+
+module.exports = function isTap(start, end, options = {}) {
```

Comment:
I thought about this too. 

Just calling is_click inside a touch handler might seem confusing to the reader but perhaps renaming it to something that an is_click and is_tap could both call would work. But I suppose the real reason I split them is because I felt that touch events do not need to gross/fine tolerance mechanism because maybe a finger is less likely to move once it has it the screen? I don't have any experience in this so it was pure hypothesis, and perhaps I am spouting ignorance.

What do you think?